stable/concourse/templates/web-deployment.yaml

```diff
@@ -281,6 +294,16 @@ spec:
             - name: CONCOURSE_DATADOG_AGENT_PORT
               value: {{ .Values.web.metrics.datadog.agentPort | quote }}
             {{- end }}
+            {{- if .Values.web.metrics.influxdb.enabled }}
+            - name: CONCOURSE_INFLUXDB_URL
+              value: {{ .Values.web.metrics.influxdb.url | quote }}
+            - name: CONCOURSE_INFLUXDB_DATABASE
+              value: {{ .Values.web.metrics.influxdb.database | quote }}
+            - name: CONCOURSE_INFLUXDB_USERNAME
+              value: {{ .Values.web.metrics.influxdb.username | quote }}
+            - name: CONCOURSE_INFLUXDB_PASSWORD
+              value: {{ .Values.web.metrics.influxdb.password | quote }}
```

Comment:
Seems kind of redundant specifying the same parameters as cli args and env vars. What's the reason for this?

Also, the password should go into a secret.

Reply:
you are right, I've corrected the changes to only set the environment variables. Also moved the influxdb password to a secret.